src/panels/lovelace/components/hui-warning.ts

```diff
@@ -35,6 +35,7 @@ export class HuiWarning extends LitElement {
         color: black;
         background-color: #fce588;
         padding: 8px;
+        overflow: hidden;
```

Comment:
We should wrap it instead of hiding? Add a `work-break`

Reply:
```suggestion
    	word-break: break-word;
```

![image](https://user-images.githubusercontent.com/18730868/94930478-13db3e00-048c-11eb-8867-4f7b28488b94.png)